package.json

```diff
@@ -16,38 +17,36 @@
   ],
   "author": "Andris Reinman <andris@kreata.ee>",
   "scripts": {
-    "build": "./scripts/build.sh",
-    "lint": "$(npm bin)/standard",
+    "build": "rm -rf ./dist/ && npm run build.types && npm run babel.compile",
```

Comment:
the `./scripts/build.sh` did too much, automatically creating a commit doesn't seem easy to work with